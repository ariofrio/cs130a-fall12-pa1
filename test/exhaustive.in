print

lookup 0
lookup 1
lookup -1
lookup -100
lookup 100
deleteMax
delete 0
delete 1
delete -1
delete -100
delete 100
print

insert 1
print
lookup 1
deleteMax
print
lookup 1
insert 1
print
lookup 1
delete 1
print
