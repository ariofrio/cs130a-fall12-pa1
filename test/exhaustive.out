

item not found
item not found
item not found
item not found
item not found
max item not present since table is empty
item not present in the table
item not present in the table
item not present in the table
item not present in the table
item not present in the table


item successfully inserted
1
1
item found
max item 1 deleted
item not found
item successfully inserted
item found
item succesfully deleted
